tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/RegionalAuthIntegrationTests.cs

```diff
@@ -205,10 +278,11 @@ public async Task AcquireTokenToGlobalEndpointThenRegionalEndpoint_UseTokenFromC
 
         private async Task<AuthenticationResult> CreateAuthenticationResultAsync(
```

Comment:
Not part of this PR, but I would maybe rename this to `GetAuthenticationResultAsync`, so that it's clearer that we're calling a method on CCA and not creating a mock auth result object?

Reply:
Updated